synfig-studio/src/gui/docks/dock_toolbox.cpp

```diff
@@ -134,6 +134,23 @@ Dock_Toolbox::~Dock_Toolbox()
 		studio::App::dock_toolbox=NULL;
 }
 
+void Dock_Toolbox::write_layout_string(string& params) const
+{
+	char num_str[6];
+	snprintf(num_str, 5, "%d", tool_box_paned->get_position());
+	params += std::string(num_str);
```

Comment:
@rodolforg Sorry, I did not understanding - why use "+=" instead of "="? ^__^"

Reply:
I don't know. lol